edocumentation-application/src/main/java/org/dcsa/edocumentation/controller/BookingSummaryController.java

```diff
@@ -27,67 +25,68 @@
 public class BookingSummaryController {
 
   private final BookingSummaryService service;
-  private final Paginator paginator;
 
   @GetMapping(path = "${spring.application.bkg-context-path}" + "/booking-summaries")
   @ResponseStatus(HttpStatus.OK)
   public List<BookingSummaryTO> getBookingSummaries(
       @RequestParam(required = false) BkgDocumentStatus documentStatus,
-      @RequestParam(required = false, defaultValue = "100") Integer limit,
-      @RequestParam(value = "sort", required = false) String sort,
+      @RequestParam(value = Pagination.DCSA_PAGE_PARAM_NAME, defaultValue = "0", required = false)
+          @Min(0)
+          int page,
+      @RequestParam(
+              value = Pagination.DCSA_PAGESIZE_PARAM_NAME,
+              defaultValue = "100",
+              required = false)
+          @Min(1)
+          int pageSize,
+      @RequestParam(value = Pagination.DCSA_SORT_PARAM_NAME, required = false) String sort,
       @RequestParam(value = "API-Version", required = false) String apiVersion,
       HttpServletRequest request,
       HttpServletResponse response) {
 
-    Sorter sorter = configureSorter();
-    Cursor cursor =
-        paginator.parseRequest(request, new CursorDefaults(limit, sorter.parseSort(sort)));
-
-    PagedResult<BookingSummaryTO> bookingSummaryResults =
-        service.findBookingSummaries(cursor, documentStatus);
-    paginator.setPageHeaders(request, response, cursor, bookingSummaryResults);
-
-    return bookingSummaryResults.content();
+    return Pagination.with(request, response, page, pageSize)
+        .sortBy(
+            sort,
+            List.of(new Sort.Order(Sort.Direction.ASC, "bookingRequestCreatedDateTime")),
```

Comment:
(Optional) Considering defaultSort and sortableField are always constant I think it would improve readability to define them as private final variables outside the method so that the sortBy method can be called with 

.sortBy(sort, defaultSort, sortableFields)

Especially since these calls becomes even more cluttered later in this PR